### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Laitenela**

### CODE REVIEW

**Comment by Polyana1997** [comment]
Чек

File: README.md [hunk 1]
```diff
@@ -1,2 +1 @@
 # FanService_Backend
-test
```
Comment (Polyana1997) [inline README.md:2]: checks

Comment (Laitenela) [reply README.md:2]: ))

Comment (Polyana1997) [reply README.md:2]: test

**Review by Polyana1997 — APPROVED** [review]
test2

**Pull request merged**